index.js

```diff
@@ -19,11 +19,11 @@ app.get('/api/v1/data/closet/', (req, res) => {
   res.status(200).json({ closetData });
```

Comment:
I don't think we are every actually just getting all closet data without filtering it but should we update the get for all closet data just to keep it in sync with the rest? (Add userID as a required param to this endpoint)